metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -289,6 +300,15 @@ class ScalametaLanguageServer(
     sourceChangeSubscriber.onNext(input)
   }
 
+  override def executeCommand(request: WorkspaceExecuteCommandRequest) = Task {
+    import WorkspaceCommand._
+    WorkspaceCommand.withNameOption(request.params.command).map {
+      case ClearIndexCache =>
+        logger.info("Clearing the index cache")
+        ScalametaLanguageServer.clearCacheDirectory()
```

Comment:
we currently clear the index, but we don't trigger re-indexing. I think it's fine for debugging to have this level of granularity, but we may want to add a separate command for triggering re-indexing.